libraries/AP_Landing/AP_Landing_Deepstall.cpp

```diff
@@ -364,6 +365,24 @@ int32_t AP_Landing_Deepstall::get_target_airspeed_cm(void) const
     }
 }
 
+bool AP_Landing_Deepstall::send_deepstall_message(mavlink_channel_t chan) const
+{
+//    CHECK_PAYLOAD_SIZE(DEEPSTALL);
```

Comment:
Do you need a pair of :eyeglasses:? :stuck_out_tongue: You missed the 2 in the CHECK_PAYLOAD_SIZE**2** macro :wink: